Libplanet/Store/LiteDBStore.cs

```diff
@@ -459,15 +472,20 @@ public long GetTxNonce(string @namespace, Address address)
 
             using (var stream = new MemoryStream())
             {
+                file.CopyTo(stream);
+
+                if (stream.Length == 0)
```

Comment:
Isn't this condition can be detected without copying `file` contents to `stream`?  IMO we could use `LiteFileInfo.Length` property.

Reply:
By the way, how could this condition happen?  Is there any code path that creates a nonce file and immediately closes it without writing anything?  Even if such a code path exists, isn't *that* code a bug, not this code?

Reply:
This condition can happen [here](https://github.com/planetarium/libplanet/commit/03caf772deef950236387d4502bd2eed9ab23e17#diff-3fe3c469228f579007de2d2703d077d9R554). After open `destFile`, you don't write anything if `currentIndex` is less than `branchPointIndex`. Do you think it is better to fix it?

Reply:
To make the code more defensive, we'd better change both side.

Reply:
Fixed to delete the file if nothing is written.